sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample3_BackRestoreResume.md

```diff
@@ -0,0 +1,46 @@
+# Checking the status of a previously started full key backup and restore
+
+This sample demonstrates how to a check the status and get the result of a previously started full key backup and restore in Azure Key Vault.
+To get started, you'll need a URI to an Azure Key Vault. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/README.md) for links and instructions.
+
+## Checking status of a full key backup
+
+Using the `KeyVaultBackupClient` and a `BackupOperation`, you can check the status and retrieve the result of a previously started `BackupOperation`. 
+For example the `Id` from a started operation on one client can be saved to persistent storage instead of waiting for completion immediately. 
+At some later time, another client can retrieve the persisted operation Id and construct a `BackupOperation` using a `KeyVaultBackupClient` and the Id 
+and check for status or wait for completion.
+
+```C# Snippet:ResumeBackupAsync
+// Construct a new KeyVaultBackupClient or use an existing one.
+KeyVaultBackupClient Client = new KeyVaultBackupClient(new Uri(keyVaultUrl), new DefaultAzureCredential());
+
+// Construct a BackupOperation using a KeyVaultBackupClient and the Id from a previously started operation.
+BackupOperation backupOperation = new BackupOperation(client, backupOperationId);
+
+// Wait for completion of the BackupOperation.
+Response<Uri> backupResult = await backupOperation.WaitForCompletionAsync();
```

Comment:
Why not make use of the implicit cast operator and just assign to `Uri backupBlobUri` now? We do that in other samples. Rarely do customers need the `Response<T>` or `Response` objects.

Reply:
Discussed offline - we have been leaning towards not using the implicit cast in samples due to UX study feedback.